operator/cmd/root.go

```diff
@@ -738,7 +738,7 @@ func (legacy *legacyOnLeader) onStart(_ cell.HookContext) error {
 	}
 
 	if legacy.clientset.IsEnabled() {
-		if err := labelsfilter.ParseLabelPrefixCfg(option.Config.Labels, option.Config.NodeLabels, option.Config.LabelPrefixFile); err != nil {
+		if err := labelsfilter.ParseLabelPrefixCfg(logging.DefaultSlogLogger, option.Config.Labels, option.Config.NodeLabels, option.Config.LabelPrefixFile); err != nil {
```

Comment:
Could we use `legacy.logger` here instead?

```suggestion
		if err := labelsfilter.ParseLabelPrefixCfg(legacy.logger, option.Config.Labels, option.Config.NodeLabels, option.Config.LabelPrefixFile); err != nil {
```

Reply:
oh, I didn't know that `legacy` already had a logger. I'll change it. I'm planning to do a final pass on all `logging.DefaultSlogLogger` once the slog conversion is almost done.